docs/developers/updates.md

```diff
@@ -26,6 +26,46 @@ function removeData(chart) {
 }
 ```
 
+## Opdating Options
+
+To update the options, mutating the options propery inplace or passing in a new options object are supported.
+
+- If the options are mutated inplace, other options properties would be preserved.
+- If created as a new object, it would be like creating a new chart with the options - old options would be discarded.
+
+```javascript
+function updateConfigByMutating(chart) {
+    chart.options.title.text = 'new title';
+}
+```
+
+```javascript
+function updateConfigAsNewObject(chart) {
+    chart.options = {
+        responsive: true,
+        title:{
+            display:true,
+            text: 'Chart.js'
+        },
+        scales: {
+            xAxes: [{
+                display: true
+            }],
+            yAxes: [{
+                display: true
+            }]
+        }
+    }
+}
+```
+
+Scales can be updated separately without changing other options.
+It is recommended to update scales by creating a new object rather than mutating the properties to be changed, unless you are confident all scales options would update as you wish - especially when upating type of the scale.
```

Comment:
Is this ok? Or is it better to just recommend update as a new object when changing type.